### PR TITLE
PT-5215 Migrate from quay.io to ECR

### PR DESCRIPTION
**Pull request by fftorres**
This PR changes references to HelloFresh private repos on quay.io to ECR repos in Concourse pipeline definition

This PR was created automatically. We checked for and updated
* `*.yml` files in `ci` and `ahoy` directories
* `*.yaml` files in `ci` and `ahoy` directories
* `Makefile`
* and `Dockerfile`

If you have references to HelloFresh private `quay.io` repos in other files, please update them and **merge this PR when you're done.**

Mit freundlichen Grüßen
Engineering Releases Team

### CODE REVIEW

**Comment by antonu17** [comment]
+1

**Comment by fftorres** [comment]
+!

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM golang:1.12
 RUN go get github.com/prometheus/prometheus/cmd/promtool
 
 # Stage 2: Usual stuff
-FROM quay.io/hellofresh/ci-ansible
+FROM 489198589229.dkr.ecr.eu-west-1.amazonaws.com/ci-ansible
 
 ARG VERSION=v2.14.3
```
Comment (harshal-shah) [inline Dockerfile:8]: This needs to be changed to helm 2.16.1

Comment (harshal-shah) [reply Dockerfile:8]: Also KUBERNETES_VERSION should change to 1.15.10

Comment (fftorres) [reply Dockerfile]: I have updated the versions, thanks! This branch is pretty old. Can I have your +1 to merge it?

**Comment by harshal-shah** [comment]
+1

**Pull request merged**